offer-manager-domain/src/test/groovy/life/qbic/portal/portlet/offers/OfferSpec.groovy

```diff
@@ -444,4 +421,58 @@ class OfferSpec extends Specification {
         externalAcademicAffiliation | 0.2
 
     }
+
+    def "Given data analysis service items, calculate the correct amount of sample-dependent discount"() {
```

Comment:
```suggestion
    def "Given #samples data analysis service items, calculate the correct amount of sample-dependent discount with factor #discountFactor"() {
```

Reply:
uh, nice one, was not aware of this feature